src/omniperf_analyze/analysis_webui.py

```diff
@@ -163,7 +166,7 @@ def generate_from_filter(
                 # update roofline for visualization in GUI
                 self.get_socs()[self.arch].analysis_setup(
                     roofline_parameters={
-                        "workload_dir": self.dest_dir,
+                        "path_to_dir": self.dest_dir,
```

Comment:
Should be `workload_dir` to reflect the run parameters used in Roofline()
```suggestion
                        "workload_dir": self.dest_dir,
```